helios.js

```diff
@@ -25905,6 +25948,13 @@ class TxBody extends CborData {
 		this.#scriptDataHash = scriptDataHash;
 	}
 
+	/**
+	 * @param {Hash} metadataHash
+	 */
+	setAuxiliaryDataHash(metadataHash) {
```

Comment:
You could probably name this `setMetadataHash`, and even use *metadata* instead of *auxiliaryData* everywhere else (we probably won't use this data structure to store anything else)